Pipfile

```diff
@@ -0,0 +1,57 @@
+[[source]]
+name = "pypi"
+verify_ssl = true
+url = "https://pypi.org/simple"
+
+[packages]
+anyjson = ">=0.3.3"
+coverage = ">=4.0.3"
+coveralls = ">=1.1"
+Django = ">=2.0.7"
+django-debug-toolbar = ">=1.9"
+django-redis-cache = ">=1.7.0"
+django-reversion = ">=2.0.6"
+django-user_agents = ">=0.3.0"
+docopt = ">=0.6.2"
+feedparser = ">=5.2.1"
+geojson = "~=1.3"
+isodate = ">=0.5.4"
+jsonfield = ">=1.0.3"
+lxml = "~=3.0"
+mysolr = "==0.8.3"
+numpy = "==1.11.0"
+"psycopg2" = "==2.7.5"
+pyparsing = "==2.1.1"
+python-dateutil = "==2.5.3"
+pytz = "==2016.4"
+rdflib = "==4.2.1"
+rdflib-jsonld = "==0.4.0"
+redis = ">=2.10.5"
+requests = ">=2.20.0"
+roman = "==2.0.0"
+scipy = "==0.17.0"
+six = "==1.10.0"
+sqlparse = ">=0.2.4"
+titlecase = "==0.9.0"
+ua-parser = ">=0.7.1"
+user-agents = ">=1.0.1"
+python-memcached = ">=1.59"
+pyproj = "==1.9.5"
+atomicwrites = "==1.2.1"
+attrs = "==18.2.0"
+more-itertools = "==4.3.0"
+"pathlib2" = "==2.3.2"
+pluggy = "==0.8.0"
+py = "==1.7.0"
+pytest = "==3.9.3"
+pytest-django = "==3.4.3"
+Pillow = "==5.2.0"
+PyYAML = ">=3.11"
+Shapely = "==1.5.15"
+Unidecode = "==0.4.19"
+uWSGI = "==2.0.17.1"
+
+[dev-packages]
+
+[requires]
+python_version = "3.6"
```

Comment:
Perhaps a test with 3.7.x to check this?

Reply:
Resolved